test/dbscan.jl

```diff
@@ -56,14 +56,17 @@ end
 
     points = [p0 p1 p2]
 
+    # FIXME The current test implementation relies on random numbers sequence being exactly the same accross all tested julia versions,
+    #       so currently it's broken. We need a new test that is not (so much) seed-dependent.
```

Comment:
How about using StableRNGs? I'm afraid if we skip these test we will forget to reenable them forever. :-/

Reply:
Thanks, StableRNGs were not on my radar.